BaseFilter/sections/general_url.txt

```diff
@@ -26,11 +26,12 @@
 ! ex. https://fiveyardlab.com/z-7131650
 /^https:\/\/[0-9a-z]{5,}\.[a-z]{2,3}\/z-[5-7]\d{6}$/$script,~third-party
 ! https://github.com/AdguardTeam/AdguardFilters/pull/175819
+/^https:\/\/www\.[a-z]{8,16}\.com\/(?:[A-Za-z]+\/)*(?:[_0-9A-Za-z]{1,20}[-.])*[_0-9A-Za-z]{1,20}\.js$/$script,third-party,match-case,header=popads-node
 /^https:\/\/www\.[a-z]{8,16}\.com\/(?:[A-Za-z]+\/)*(?:[_0-9A-Za-z]{1,20}[-.])*[_0-9A-Za-z]{1,20}\.js$/$script,third-party,match-case,header=link:/adsco\.re\/>;rel=preconnect/
-://www.*.com/*.css|$script,third-party,header=link:/adsco\.re\/>;rel=preconnect/
-://www.*.com*/images/*.min.js|$script,third-party,header=link:/adsco\.re\/>;rel=preconnect/
-://www.*.com/css/$script,third-party,header=link:/adsco\.re\/>;rel=preconnect/
-://www.*.com/js/css/$script,third-party,header=link:/adsco\.re\/>;rel=preconnect/
+://www.*.com/*.css|$script,third-party,header=link:/\/>;rel=preconnect/
+://www.*.com*/images/*.min.js|$script,third-party,header=link:/\/>;rel=preconnect/
+://www.*.com/css/$script,third-party,header=link:/\/>;rel=preconnect/
+://www.*.com/js/css/$script,third-party,header=link:/\/>;rel=preconnect/
```

Comment:
@Yuki2718 Does not look safe. Could you please show a value of `link`?

Reply:
`<https://cqnqrcddbacr.com/>;rel=preconnect` for `https://www.cqnqrcddbacr.com/mfirebase.css`. These rules are mostly safe even without `$header`, the only known FP is Principal Financial Group https://github.com/uBlockOrigin/uAssets/issues/22815.

<details>
<summary>Screenshot</summary>

![header](https://github.com/AdguardTeam/AdguardFilters/assets/58900598/1d0a590a-4966-418f-9b55-d885c6943e39)

</details>

Reply:
Hmm. I see `Popads-Node` header on the screenshot. What about to use it?
`link:****;rel=preconnect` is used also for legal purposes.

Reply:
The old pattern without `Popads-Node` is still in use. So if we use this, we have to double up the number of rules.

Reply:
Okay.